ui/components/component-library/tag/tag.scss

```diff
@@ -0,0 +1,6 @@
+.tag {
+  height: auto;
+  width: max-content;
+  padding-top: 2px;
+  padding-bottom: 2px;
```

Comment:
Because the border is inline in the figma file I think we want this to be `1px`

```suggestion
  padding-top: 1px;
  padding-bottom: 1px;
```

<img width="1364" alt="Screen Shot 2022-10-06 at 6 28 21 PM" src="https://user-images.githubusercontent.com/8112138/194447207-bb477237-09aa-4219-ab7d-b2a63834199b.png">

![Screen Shot 2022-10-06 at 6 29 00 PM](https://user-images.githubusercontent.com/8112138/194447208-8b0fcac4-6525-4f68-9675-1c9ef814e007.png)

Reply:
Updated!